src/styles/utiles/variable.scss

```diff
@@ -0,0 +1,4 @@
+$main-color-font: #060b35;
+$hover-color-font:#00acdc;
+$main-font-size: 12px;
+$second-font-size: 10px;
```

Comment:
Consider renaming the variable to $secondary-font-size for better readability and consistency.